bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jasmine-utils",
-  "version": "0.2.3",
+  "version": "0.2.9",
```

Comment:
Can you remove this change please (the version number will be updated when the release will be made)?